Sources/Subprocess/IO/Input.swift

```diff
@@ -33,19 +33,18 @@ import FoundationEssentials
 
 // MARK: - Input
 
-/// `InputProtocol` defines the `write(with:)` method that a type
-/// must implement to serve as the input source for a subprocess.
+/// InputProtocol defines the `write(with:)` method that a type must
+/// implement to serve as the input source for a subprocess.
```

Comment:
```suggestion
/// InputProtocol defines a type that serves as the input source for a subprocess.
///
/// The protocol defines the `write(with:)` method that a type must
/// implement to serve as the input source.
```

In general, abstracts for symbols should be a single sentence that doesn't use code voice (symbol references or monospaced fonts). Breaking it up with a line break in between drops the detail about the method that must be provided (which the protocol conformance enforces) into the discussion section for the type. The abstract (that first line) is propagated and shown alongside the content anywhere the symbol is referenced.